src/static/riot/competitions/form/_leaderboard.tag

```diff
@@ -346,10 +346,10 @@
         self.on("update", function () {
             // Force refresh of dropdown on update, otherwise they don't show new elements and
             // don't reflect new names
-            let dropdown = $(".dropdown", self.root)
-            dropdown.dropdown("refresh")
+            let leaderboard_dropdown = $(".dropdown", self.root)
+            leaderboard_dropdown.dropdown("refresh")
```

Comment:
ahhh did this fix it? or just one of the pieces in the attempt to fix?

Reply:
Just one of the steps. didn't do anything but I really didn't like just `dropdown` as a variable name anyway. Figured this would clear it up/make _sure_ this wouldn't cause an issue elsewhere.